packages/garbo/src/fights.ts

```diff
@@ -525,9 +524,17 @@ export function dailyFights(): void {
         const underwater = location.environment === "underwater";
         const shouldCopy = get("_badlyRomanticArrows") === 0 && !underwater;
 
-        const bestCopier = $familiars`Obtuse Angel, Reanimated Reanimator`.find(
-          have,
-        );
+        // use obtuse angel if have + have quake of arrows, otherwise reanimator
+        // quake of arrows is PvP-stealable and costs ~50k, so don't assume we have it
+        let bestCopier: Familiar | undefined;
+        if (
+          have($familiar`Obtuse Angel`) &&
+          retrieveItem($item`quake of arrows`)
```

Comment:
I don't think this is the best solution because `retrieveItem` may unequip the quake of arrows unnecessarily.

Reply:
I think this should sort it?

Reply:
Yeah!